docs/groovy-script/mods/academy/imag_fusor.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Imag Fusor"
+titleTemplate: "AcademyCraft | CleanroomMC"
+description: "Converts one item and defined amount of Imag phase liquid into one item."
+source_code_link: "https://github.com/yor42/AcademyCraft/blob/master/src/main/java/cn/academy/support/groovyscript/modules/ImagFusor.java"
+---
+
+# Imag Fusor (AcademyCraft)
+
+## Description
+
+Converts one item and defined amount of Imag phase liquid into one item.
```

Comment:
i believe the name is `Imag Phase Liquid`, so the `p` and `l` should be capitalized. although perhaps just the `p`? would lean both